helpers.php

```diff
@@ -0,0 +1,18 @@
+<?php 
+
+function unique_user_name($db, $username){
+    $stmt = $db->prepare('select count(name) as number from user where name = ?');  
+    if ($stmt === false) {
+        return $this->renderer->render($response, '/register.php', ['err' => $this->database->lastErrorMsg(),'err2' =>'Connection with server failed. Please try again later']);
```

Comment:
Nope. Its not the function role to render a view. Not sure this should work at all….